src/Simplex/Chat/Store/SQLite/Migrations/M20250526_short_links.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Store.SQLite.Migrations.M20250526_short_links where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20250526_short_links :: Query
+m20250526_short_links =
+  [sql|
+ALTER TABLE contacts ADD COLUMN inv_conn_req_to_connect BLOB;
+ALTER TABLE contacts ADD COLUMN addr_conn_req_to_connect BLOB;
```

Comment:
I don't think this is necessary to have two fields here, we have type that can accomodate both, and you can build logic off subtypes - that is what subtype of link is stored here would determine what happens. Some additional field may still be necessary if we need to differentiate something in UI.